unit-tests/live/metadata/test-metadata-alive.py

```diff
@@ -0,0 +1,112 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+# test:device each(D500*)
+
+import pyrealsense2 as rs
+from rspy import test
+
+
+def close_resources(s):
+    """
+    Stop and close a sensor
+    :param s: sensor of device
+    """
+    if len(s.get_active_streams()) > 0:
+        s.stop()
+        s.close()
+
+
+def is_contain_profile(profiles: dict, new_profile) -> bool:
+    """
+    Check if a given stream type exists in a dictionary
+    :profiles: Dictionary of profiles and sensors
+    :new_profile: Profile that we want to check if it is in a dictionary
+    :returns: True if a type of profile is already in the list otherwise False
+    """
+    if new_profile:
+        for pr in profiles.keys():
+            if pr.stream_type() == new_profile.stream_type():
+                return True
+    return False
+
+
+def is_frame_support_metadata(frame, metadata) -> bool:
+    """
+    Check if a metadata supporting by a frame
+    :param frame: a frame from device
+    :param metadata: a metadata of frame
+    :return: true if metadata supporting by frame otherwise false
+    """
+    return frame and frame.supports_frame_metadata(frame_metadata=metadata)
```

Comment:
What does `frame_metadata=metadata` means?

Reply:
We have another test that check `supports_frame_metadata`, maybe this 2 tests can be merged?

Reply:
> What does `frame_metadata=metadata` means?

In this line, I check if a received frame contains/supports received metadata (by using a function from frame.py).

Reply:
> We have another test that check `supports_frame_metadata`, maybe this 2 tests can be merged?

Are you talking about [test-rgb-options-metadata-consistency.py](unit-tests/live/options/test-rgb-options-metadata-consistency.py)?